ui/components/component-library/picker-network/picker-network.tsx

```diff
@@ -44,20 +46,23 @@ export const PickerNetwork: PickerNetworkComponent = React.forwardRef(
         backgroundColor={BackgroundColor.backgroundAlternative}
         alignItems={AlignItems.center}
         paddingLeft={2}
-        paddingRight={4}
+        paddingRight={2}
         gap={2}
         borderRadius={BorderRadius.pill}
         display={Display.Flex}
         {...(props as BoxProps<C>)}
       >
-        <AvatarNetwork
-          className="mm-picker-network__avatar-network"
-          src={src}
-          name={label}
-          size={AvatarNetworkSize.Xs}
-          {...avatarNetworkProps}
-        />
-        <Text as="span" ellipsis variant={TextVariant.bodySm} {...labelProps}>
+        {avatarComponent ?? (
+          <AvatarNetwork
+            className="mm-picker-network__avatar-network"
+            src={src}
+            name={label}
+            size={AvatarNetworkSize.Xs}
+            {...avatarNetworkProps}
+          />
+        )}
+
```

Comment:
Picker Network is for changing a network so I am not sure why we are making it conditional.

Reply:
Refactored this so an override for the avatar is no longer required. The PickerNetwork for bridge has multiple stacked network icons so we need to be able to render an `AvatarGroup`. Will add a story for this as well